### PR TITLE
mjml-react: Add missing table props

### PR DESCRIPTION
**Pull request by adamth**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://documentation.mjml.io/#mj-table
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/mjml-react/index.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for mjml-react 1.0
+// Type definitions for mjml-react 2.0
```
Comment (henrinormak) [inline types/mjml-react/index.d.ts:1]: @adamth are the types backwards compatible, i.e no breaking changes for the types between 1.0 and 2.0?

File: types/mjml-react/index.d.ts [hunk 2]
```diff
@@ -382,6 +382,9 @@ export interface MjmlTableProps {
     width?: string | number | undefined;
     tableLayout?: 'auto' | 'fixed' | 'initial' | 'inherit' | undefined;
     align?: 'left' | 'right' | 'center' | undefined;
+    role?: 'presentation' | 'none';
+    border?: React.CSSProperties['border'] | undefined;
+    cssClass?: string | undefined;
```
Comment (henrinormak) [inline types/mjml-react/index.d.ts:387]: I would suggest using the already existing `ClassNameProps` interface in the intersection type below, that way this prop should not be needed.

`export class MjmlTable extends React.Component<MjmlTableProps & PaddingProps & ClassNameProps> { }`

File: types/mjml-react/index.d.ts [hunk 3]
```diff
@@ -382,6 +382,9 @@ export interface MjmlTableProps {
     width?: string | number | undefined;
     tableLayout?: 'auto' | 'fixed' | 'initial' | 'inherit' | undefined;
     align?: 'left' | 'right' | 'center' | undefined;
+    role?: 'presentation' | 'none';
+    border?: React.CSSProperties['border'] | undefined;
```
Comment (henrinormak) [inline types/mjml-react/index.d.ts:386]: Does the table support all of the border properties? i.e could this be replaced by the component also allowing for all `BorderProps`?

**Review by henrinormak — CHANGES_REQUESTED** [review]
Hoping this could use the existing interfaces for `BorderProps` and `ClassNameProps`. If not, let me know and I'll re-review

File: types/mjml-react/index.d.ts (hunk 3 above)
Comment (adamth) [reply types/mjml-react/index.d.ts:386]: It looks like the only supported prop is `border`, oddly enough 
https://github.com/mjmlio/mjml/blob/89fe24c0cb716c9b9531974eb49e2f251a46b22d/packages/mjml-table/src/index.js#L13

File: types/mjml-react/index.d.ts (hunk 1 above)
Comment (adamth) [reply types/mjml-react/index.d.ts:1]: It appears that the only change for the major version bump was updating the babel runtime https://github.com/wix-incubator/mjml-react/commit/7894e20e52db6340e8bad2cbb731f6a3f2430db4
So no breaking changes 🎉

File: types/mjml-react/index.d.ts (hunk 2 above)
Comment (adamth) [reply types/mjml-react/index.d.ts]: Oops, missed that one. Have updated

**Review by henrinormak — APPROVED** [review]

**Comment by henrinormak** [comment]
Ready to merge

**Pull request merged**